html/tests/alerts.tests.js

```diff
@@ -26,7 +27,7 @@ describe('Alert tests', () => {
 
   it('should hide alert', () => {
     dismissAlert(container);
-    expect(container.classList.contains('sprk-u-Display--none')).toBe(true);
+    expect(container.classList.toString()).toBe('sprk-c-Alert sprk-c-Alert--is-hidden');
```

Comment:
I made these class checks for the Alert more strict by switching it to check the entire class string.